rootless.md

```diff
@@ -0,0 +1,54 @@
+# Shortcomings of RootLess Podman
+
+* Not a ton of “How To” documentation or documentation in general
+* If you don’t have /etc/subuid and /etc/subgid setup for a user, then it can easily fail
+* This causes issues on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
+** Can not create containers that bind to ports < 1024
+* No Cgroup Support (hopefully fixed when Cgroups V2 happens).
+* Can not share container images with CRI-O or other users
+* Difficult to use Additional Stores for sharing content
+* Does not work on NFS Homedirs or noexec/nodev homedirs
+* Can not use Overlayfs driver, but does support fuse-overlayfs
+** Does not perform as well as root podman with Kernel based Overlay
+* Only other supported driver is VFS.
+* No KATA Container support ???
+* Difficult to use multiple different User Namespaces
+* No CNI Support
+* Cannot use ping (Can be fixed by setting sysctl on host)
+* Requires new shadow-utils (not found in older (RHEL7/Centos7 distros) Should be fixed in RHEL7.7 release
+* Some commands don’t work.
```

Comment:
This subsection resolves badly, the duplicate below looks good though

Reply:
Yes don't know how that happened.
Fixed,